src/Microsoft.Tye.Core/Serialization/ConfigIngressParser.cs

```diff
@@ -144,6 +146,17 @@ private static void HandleIngressBindingMapping(YamlMappingNode yamlMappingNode,
 
                         binding.Port = port;
                         break;
+                    case "ip":
+                        if (YamlParser.GetScalarValue(key, child.Value) is string ipString
+                            && (IPAddress.TryParse(ipString, out var ip) || ipString == "*" || ipString.Equals("localhost", StringComparison.OrdinalIgnoreCase) ))
+                        {
+                            binding.IPAddress = ip == IPAddress.Loopback || ip == IPAddress.IPv6Loopback ? null : ipString;
```

Comment:
Why ignore the user's explicit setting if it happens to be the loopback address?